gossip/src/gossip_service.rs

```diff
@@ -193,37 +197,40 @@ pub fn discover(
     ))
 }
 
-/// Creates a ThinClient by selecting a valid node at random
+/// Creates a TpuClient by selecting a valid node at random
 pub fn get_client(
```

Comment:
Just calling out that this method and `get_multi_client` are public apis, so these changes are breaking. We need to be careful about such things. In this case, since crates.io reports no dependents of solana-gossip except for other monorepo crates and jito, I'm okay with it. However, what do you think about adding `deprecated` tags in v1.18 to be thorough?

Reply:
ohhhhh shoot good point. what exactly do you mean? Make a separate PR that deprecates `get_client` and `get_multi_client` and then backport it to v1.18? And then after that we merge this PR?

Reply:
> Make a separate PR that deprecates `get_client` and `get_multi_client` and then backport it to v1.18? And then after that we merge this PR?

Yes, exactly! The note on `get_client` could say the interface will change?